connexion/spec.py

```diff
@@ -207,17 +244,21 @@ def base_path(self, base_path):
 
     @classmethod
     def _validate_spec(cls, spec):
```

Comment:
This method is now the same in both classes, so we should be able to lift this to the base `Specification` class.